contracts/access/README.adoc

```diff
@@ -32,8 +32,12 @@ This directory provides ways to restrict who can access the functions of a contr
 
 {{IAuthority}}
 
+{{IAccessManager}}
+
 {{AccessManager}}
 
+{{IAccessManaged}}
```

Comment:
If we'll include AccessManaged we have to add NatSpec there as well

Reply:
done